clothing/views.py

```diff
@@ -1,3 +1,18 @@
-from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views.generic import CreateView
+from django.utils.translation import ugettext as _
+from django.views.generic import TemplateView
 
-# Create your views here.
+from clothing.forms import OrderClothingForm
+from clothing.models import Order
+
+
+class OrderClothingView(CreateView):
```

Comment:
I think it would be better to have a multi step order process which make use of [django.contrib.formtool](https://django-formtools.readthedocs.io/en/latest/). With the following steps:
1. auth against the e-mail address
2. Place the order
3. Summary of the order

In 2 there should be shown the orders already given and the possibility to change or cancel them.